app/src/main/res/values/strings.xml

```diff
@@ -12,4 +12,7 @@
   <string name="cellular_data_alert_dialog_title">Currently on Cellular Data</string>
   <string name="cellular_data_alert_dialog_description">Streaming audio may use a lot of cellular data.</string>
   <string name="cellular_data_alert_dialog_checkbox">Don\'t show this message again</string>
+  <string name="topic_train_master_these_skills">Master These Skills</string>
+  <string name="topic_train_skills_description">Select or unselect skills that you want to practice</string>
```

Comment:
you can add full stop to the long messages

Reply:
The mock does not contain any `.` thats why, i kept it as it is.